app/code/core/Mage/AdminNotification/Model/Resource/Inbox.php

```diff
@@ -1,4 +1,5 @@
 <?php
+
```

Comment:
I would love this PR it it wasn't for this empty line before the copyright docblock

Reply:
This empty line is part of code sniffs :/

Reply:
sure I know, but mmmm it seems symfony uses this notation but it it's the first time that I see it

Reply:
Of course it can but disabled, but ... IMHO we should just exclude sniffs that are required for code we cant fix, (properties/methods with underscore, ...). I know adding a blank line looks wierd ...